src/Enum/Type.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\DpdPlPickupServicesBundle\Enum;
+
+use MabeEnum\Enum;
+
+class Type extends Enum
+{
+    public const STANDARD = 100;
+    public const NETWORK = 200;
+    public const FULL_TIME = 300;
```

Comment:
Not sure about naming for the last two: these are "punkt sieciowy" and "punkt etatowy" in Polish, I'm on a look out for better translations

Reply:
"punkt sieciowy" is not something like "chain of point"? Similar to `chain of stores`?
Of course it is guesswork

Reply:
Changed in e5561fea90955d7b53dfdc544d29d0388d111489